crd/hack/update-codegen.sh

```diff
@@ -112,5 +112,5 @@ codegen_for () {
           ${COMMON_FLAGS}
 }
 
+codegen_for network v1
```

Comment:
Let's leave the generate_config statement and add codegen_for

Reply:
codegen_for looks like it calls generate_config which is why I removed it.

Reply:
oh yeah makes sense!